src/notebook/plugin.ts

```diff
@@ -68,6 +68,7 @@ const cmdIds = {
   restartRunAll: 'notebook:restart-runAll',
   switchKernel: 'notebook:switch-kernel',
   clearAllOutputs: 'notebook:clear-outputs',
+  killAndExit: 'notebook:halt-and-close',
```

Comment:
Absolutely! I originally chose killAndExit to match classic Jupyter, but I agree this makes more sense here and keeps things more consistent. Thanks @blink1073!